pandas/tests/extension/base/interface.py

```diff
@@ -29,6 +29,43 @@ def test_can_hold_na_valid(self, data):
         # GH-20761
         assert data._can_hold_na is True
 
+    def test_contains(self, data):
+        # GH-37867
```

Comment:
can you put a brief comment here on this discussion we are having (e.g. to distinguuish between different nans and so on ), for a future reader